### ISSUE TITLE
Adding Bluetooth , audio, mic well compatibility with it 

### ISSUE THREAD

**Issue by beastbroak30** [micropython/micropython#15437]
### Description

Ya there are many for audio mic but it's like too but bluethooh with headphone jack in micropython no it not present

### Code Size

Make it conscise workable without sd card 

### Implementation

I intend to implement this feature and would submit a Pull Request if desirable

### Code of Conduct

Yes, I agree

**Comment by jasmohan-narula** [comment]
@beastbroak30 If you are able to add support for Bluetooth Audio into Micropython, please go for it.

Even I would love to use micropython for Bluetooth audio projects.

**Comment by beastbroak30** [comment]
Ya actually I working on it and like issue is that microphone like inmp441 need a buffer to store the audio header file and moreover the playing of audio in micropython using analog like a hectic thing I working on the code as to implement the file of audio ,recording playing as a default lib to be put in a mcu

**Comment by jonnor** [comment]
Any update on this? I am considering moving this issue to a discussion, unless it is actively worked on.

**Comment by beastbroak30** [comment]
yup actually the i succeeded in adding inmp441 support in rpi pico and
esp32 using the micropython via a github repo   and other forums i will
soon update in 2 months , Right now my  high school exams are going on
